### ISSUE TITLE
Camera can be moved very far on the team map

### ISSUE THREAD

**Issue by gio3k** [Small-Fish-Dev/small-fish-dev.github.io#12]
https://github.com/user-attachments/assets/c6a9a8c3-d62c-45c9-8b32-67a9ca0a839b

**Comment by matekdev** [comment]
Yeah adding bounds checking was annoying because of zooming in and out. If someone wants to figure it out feel free to.
